docs/source/overview.md

```diff
@@ -1,21 +1,28 @@
 # Overview
 
-This documentation will refer to two roles that infrastructure providers can play:
+(Livepeer)[https://livepeer.org] is a live video streaming network protocol that is fully decentralized, highly scalable, crypto token incentivized, and results in a solution which is cheaper to an app developer or broadcaster than using traditional centralized live video solutions.
```

Comment:
cc @dob to comment on the language to use here

Reply:
How about...

Livepeer is an open video infrastructure. It enables video application developers to power their streaming applications reliably at scale at a fraction of the cost traditional centralized live video solutions. It is a fully decentralized, highly scalable, crypto token incentivized, and built in an open source manner.

FWIW both the proposed copy here, and the copy in this comment are ok, so if you prefer one over the other, or a combination of the two, based on surrounding content, that's fine.

Reply:
This is straight from go-livepeeer's README (first paragraph) as I had no inspiration, I can change it in both places if you want ?

Reply:
> This is straight from go-livepeeer's README (first paragraph) as I had no inspiration, I can change it in both places if you want ?

Ah I see. Tagged Doug in case there was particular language that we've been using elsewhere. If this is the language used in go-livepeer, I'm fine with sticking with this for now.

> Are you okay with the language used in the "broadcaster" section in "overview" that one I wrote myself.

LGTM!